spec/lib/whitehall_importer/change_history_spec.rb

```diff
@@ -0,0 +1,81 @@
+RSpec.describe WhitehallImporter::ChangeHistory do
+  let(:first_edition) do
+    build(:whitehall_export_edition,
+          state: "superseded",
+          revision_history: [build(:whitehall_export_revision_history_event,
+                                   state: "published",
+                                   created_at: Time.zone.now.last_year.rfc3339)])
+  end
+
+  it "returns change history for the published edition" do
```

Comment:
This test can probably be split in two where in the first we just want to test that this method returns some change history and then a second test where we test how the number we pass into for limits the results returned.

Reply:
I've split the test up into 4, testing each point of the import for each edition. This seems a bit OTT but gives a nice test case which tests a semi realistic situation. What do you think?